test/fixtures/linter/rules/NoGlobals/FormatterGlobalBindingString.js

```diff
@@ -0,0 +1,18 @@
+sap.ui.define(
+	["sap/m/Input", "ui5/walkthrough/model/formatter"],
+  	(Input, formatter) => {
+		// The following two cases using global notations should be detected:
+		const input = new Input({
+			value: "{ path: 'invoice>Status', formatter: 'ui5.walkthrough.model.formatter.statusText' }"
+		});
+		input.applySettings({
+			value: "{ path: 'invoice>Status', formatter: 'ui5.walkthrough.model.formatter.statusText' }",
+		});
+
+		// Although the formatter property does not look like a global notation,
+		// it should still be detected if it is a string:
+		const input2 = new Input({
+			value: "{ path: 'invoice>Status', formatter: 'formatter.statusText' }"
```

Comment:
In this case, it's clear that there cannot be fromatter like this:
`"{ path: 'invoice>Status', formatter: formatter.statusText }"`, (no quotes around `formatter.statusText`) because the BindingParser/ExpressionParser will throw an exception.
Therefore, the current state is the only possible way to define this.

Why do we need to report this case, given that the `formatter` is imported as a module?

The only reasonable answer would be that the `formatter` will not be supported when defining a string binding.

@codeworrior do I miss something here?

Reply:
To avoid misunderstandings, we're talking about the following case with quotes around the formatter _value_ and where formatter is the import name of the corresponding module:
```js
sap.ui.define(["some/formatter", ...], function(formatter, ...) {
    new Input({
        value: "{ path: 'invoice>Status', formatter: 'formatter.statusText' }"
    });
});
```
This does NOT work at runtime. The local variable "formatter" (introduced by the sap.ui.define call) is visible only within the scope of the module factory function. The framework code (which parses the binding string) is in a different scope and does not know this variable nor does it have access to it. So, for the framework code, "formatter" looks like any other global name and that's why it should be reported.

Reply:
I suggested to add this as a fixture mainly for ourselves, so that we don't think this would be a valid usage of a formatter in a binding string.

Reply:
> The only reasonable answer would be that the formatter will not be supported when defining a string binding.

If you change this slightly to 
> The only reasonable recommendation would be that the formatter will no longer be supported in UI5 2.0 when defining a string binding.

then I agree.

Reply:
Now, I get it! Thanks a lot!